Purchases/Attribution/RCAttributionTypeFactory.h

```diff
@@ -42,10 +42,16 @@ NS_SWIFT_NAME(AttributionTypeFactory)
 @interface RCAttributionTypeFactory : NSObject
 
 - (Class<FakeAdClient> _Nullable)adClientClass;
-- (Class<FakeATTrackingManager> _Nullable)atTrackingManagerClass;
+- (Class<FakeATTrackingManager> _Nullable)atTrackingClass;
```

Comment:
just in case they're looking for "ATTrackingManager" ignoring case

Reply:
I hope they are not looking for atttracking haha. It is crazy we have to deal with this 🤦

Reply:
I missed this one, but I'm renaming it just in case